ambry-replication/src/main/java/com/github/ambry/replication/ReplicaThread.java

```diff
@@ -252,7 +252,8 @@ NetworkClient getNetworkClient() {
    * Empty for current ReplicaThread but can be re-written in other cases to track progress.
    * @param remoteReplicaInfo
    */
-  protected void logReplicationCaughtUp(RemoteReplicaInfo remoteReplicaInfo) {
```

Comment:
nit: update javadocs for this method.

Reply:
This method's signature was changed but the javadocs is still not updated. It still has only one `@param` . Also no description for `remoteReplicaInfo`.